pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.black]
+skip-string-normalization = true
+
+[tool.flake8]
+show-source = true
+max-line-length = 99
+application-import-names = "wazo_call_logd"
+ignore = [
+  "E203", # whitespace before ':'
+  "W503", # line break before binary operator
+  "E501", # line length, too many tests have long strings that autoformatting won't touch
```

Comment:
That's what I did in the projects I added it too. I removed the ignore for line length and ignored or fixed the lines in question. If a specific file or module is particularly problematic it can be ignored.

Reply:
Many modules are particularly problematic such as most integration tests and many unit tests. I'd prefer not spending time modifying lots of code for this in the context of this PR. Removing this ignore seems out of scope and low value.

Reply:
Why don't you just ignore it for tests then? That seems reasonable.

Reply:
Is there a way to ignore specific warnings for whole module namespaces? I see only a way to ignore per-file, which has two issues:
* significantly more flake8 configuration to list all of those files
* the list has to be maintained along with adding and removing relevant files

At that point I'll go ahead and add `# noqa: E501` in each of those files.

Reply:
But keeping the global `E501` ignore would maintain the PR lightweight and simple, and keep the historical behavior(at least for the purpose of a first pre-commit version which will be updated later on)

Reply:
@DrPyser yes, you can do wildcards in the per-file-ignores. So, for example `tests/*: E501`. So it's not a lot og configuration.

Reply:
Oh # noqa: {error code} doesn't work at top of file. https://stackoverflow.com/questions/48153886/flake8-ignore-specific-warning-for-entire-file
Anyway, adding those `per-file-ignores` then.

Reply:
> @DrPyser yes, you can do wildcards in the per-file-ignores. So, for example `tests/*: E501`. So it's not a lot og configuration.

oh really? Tried it, didn't seem to work. I'll try again.

Reply:
Whelp you're right, done.